ansible/deploy_frontend_only.yml

```
@@ -0,0 +1,126 @@
+      file:
+        path: "{{ frontend_dest }}"
+        state: directory
+        owner: www-data
+        group: www-data
+        mode: '0755'
+
+    - name: Remove existing frontend files
+      file:
+        path: "{{ frontend_dest }}"
+        state: absent
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)